### PR TITLE
ToCode Solution: תרגול קבצים

### PR DESCRIPTION
**Pull request by idisis**

### CODE REVIEW

File: 08_files_lab/01.py [hunk 1]
```diff
@@ -1,6 +1,19 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+import argparse
```
Comment (ynonp) [inline 08_files_lab/01.py]: You accidentally submitted the same solution twice

Comment (ynonp) [inline 08_files_lab/01.py:1]: You accidentally submitted the solution to the wrong exercise here. Should take 2 file named and append the second file's contents to the end of the first one

File: 08_files_lab/02.py [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+
+with open(args.a, 'r') as a:
+  with open(args.b, 'r') as b:
+    with open(args.o, 'w') as o:
+        for line_tuple in izip_longest(a, b):
+            for line in line_tuple:
+                if line != None:
+                   o.write(line)
+                   if line[-1] != '\n':
+                     print >> o, ""
```
Comment (ynonp) [inline 08_files_lab/02.py:18]: 👍

File: 08_files_lab/01.py (hunk 1 above)
Comment (idisis) [inline 08_files_lab/01.py:1]: Sorry, I just forgot to update the description of the script in the argparse...

File: 08_files_lab/01.py [hunk 3]
```diff
@@ -16,4 +16,4 @@
         for line in b:
             o.write(line)
```
Comment (ynonp) [inline 08_files_lab/01.py:17]: Cool. Another idea - you can replace the if with a single: `print >>o, line.rstrip()`, so you cover both cases

**Pull request closed without merging**